### PR TITLE
NMS-14913: Enlink daemon doc restructure

### PR DESCRIPTION
**Pull request by mmahacek**
This PR is moving content between sections.  There aren't any changes to the actual content.

### All Contributors

* [x] Have you read our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [X] Have you (electronically) signed the [OpenNMS Contributor Agreement](https://cla-assistant.io/OpenNMS/opennms)?

### External References

* JIRA (Issue Tracker): http://issues.opennms.org/browse/NMS-14913

### CODE REVIEW

File: docs/modules/reference/pages/daemons/daemon-config-files/enlinkd.adoc [hunk 1]
```diff
@@ -3,6 +3,24 @@
 = Enlinkd
 
 The enhanced link daemon collects xref:operation:topology/enlinkd/introduction.adoc[topology information] via SNMP to determine neighboring nodes.
+The daemon essentially asks each device the following question: "What is the network topology from your point of view?"
+The enlinkd discovery process attempts to discover bridge domain links with the data coming from all collected bridge forwarding tables.
+The enlinkd updaters correlate all collected topology data to generate a global topology layout of your network.
+
+.Enlink configuration file overview
```
Comment (Bonrob2) [inline docs/modules/reference/pages/daemons/daemon-config-files/enlinkd.adoc:10]:
```suggestion
.Enlinkd configuration file overview
```

File: docs/modules/reference/pages/daemons/daemon-config-files/enlinkd.adoc [hunk 2]
```diff
@@ -14,3 +32,76 @@ The enhanced link daemon collects xref:operation:topology/enlinkd/introduction.a
...
 |===
+
+
+== Enlinkd configuration
+
+If multiple protocols are enabled, links will be discovered for each enabled discovery protocol.
+The topology web UI visualizes links for each discovery protocol.
+For example, if you start CDP and LLDP discovery, the web UI visualizes a CDP link and an LLDP link.
+
+Restart {page-component-title} or just the enlinkd daemon to apply configuration changes.
```
Comment (Bonrob2) [inline docs/modules/reference/pages/daemons/daemon-config-files/enlinkd.adoc:43]: The 
enlinkd configuration file over table says restart is not required, but this text indicates the opposite.

File: docs/modules/reference/pages/daemons/daemon-config-files/enlinkd.adoc [hunk 3]
```diff
@@ -14,3 +32,76 @@ The enhanced link daemon collects xref:operation:topology/enlinkd/introduction.a
...
+[cols="1,3,1"]
+|===
+| Attribute
+| Description
+| Default
+
+3+| *Integer*
+
+| threads
+| Number of parallel threads collectors, updaters, and discovery use.
```
Comment (Bonrob2) [inline docs/modules/reference/pages/daemons/daemon-config-files/enlinkd.adoc:56]:
```suggestion
| Number of parallel thread collectors, updaters, and discovery use.
```
"threads" sounds wrong to me, but if it's common use, ignore my comment.

**Review by Bonrob2 — CHANGES_REQUESTED** [review]
Minor changes. Thanks!

File: docs/modules/reference/pages/daemons/daemon-config-files/enlinkd.adoc (hunk 2 above)
Comment (mmahacek) [reply docs/modules/reference/pages/daemons/daemon-config-files/enlinkd.adoc:43]: This is a wording issue that plagues all the daemon files. A restart of the full horizon/Meridian server isn't required to apply changes, just a reload of the daemon.

File: docs/modules/reference/pages/daemons/daemon-config-files/enlinkd.adoc (hunk 3 above)
Comment (mmahacek) [reply docs/modules/reference/pages/daemons/daemon-config-files/enlinkd.adoc:56]: It's the number of threads used by collectors, updaters, and discovery. I'll tweak the wording to clarify.

File: docs/modules/reference/pages/daemons/daemon-config-files/enlinkd.adoc (hunk 2 above)
Comment (Bonrob2) [reply docs/modules/reference/pages/daemons/daemon-config-files/enlinkd.adoc:43]: We can consider rephrasing if we go through the section again. It's fine for now.

File: docs/modules/reference/pages/daemons/daemon-config-files/enlinkd.adoc (hunk 3 above)
Comment (mmahacek) [reply docs/modules/reference/pages/daemons/daemon-config-files/enlinkd.adoc:56]:
```suggestion
| Number of parallel threads for enlinkd collector, updater, and discovery tasks.
```

**Review by Bonrob2 — APPROVED** [review]
LGTM, thanks!

**Pull request merged**